dashboard/app/models/script.rb

```diff
@@ -87,7 +87,8 @@ def starting_level
   # indexed by both id and name. This is cached both in a class
   # variable (ie. in memory in the worker process) and in a
   # distributed cache (Rails.cache)
-  @@script_cache = nil
+  cattr_accessor :script_cache
```

Comment:
ok, now the original seems to be working correctly anyway when I tested it again. i'll remove this change~